package.json

```diff
@@ -29,7 +29,8 @@
     "react-markdown": "^9.0.1",
     "react-router": "^6.22.0",
     "react-router-dom": "^6.22.0",
-    "uuidv4": "^6.2.13"
+    "uuidv4": "^6.2.13",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz"
```

Comment:
Depending on package repositories is preferred to direct listing from vendor.
Version 0.18.5 is the latest vetted version.